superdesk/storage/async_storage/media_storage.py

```diff
@@ -0,0 +1,265 @@
+# -*- coding: utf-8; -*-
```

Comment:
It might also be good to have this module under `superdesk.core` somewhere, to differentiate between the old and the new.

Reply:
done